lib/src/util/rem_util.dart

```diff
@@ -26,8 +26,19 @@ import 'package:over_react/src/util/css_value_util.dart';
 import 'package:over_react/react_dom.dart' as react_dom;
 import 'package:platform_detect/platform_detect.dart';
 
+// The computed font size of the HTML node isn't reliable in Chrome when the page is refreshed while zoomed.
+// Instead, measure a container to determine how big 1rem is.
```

Comment:
Can we link to https://bugs.chromium.org/p/chromium/issues/detail?id=1043349 ? Can do in a follow up PR if desired, so long as it gets in there.